src/sentry/api/bases/__init__.py

```diff
@@ -1 +1,6 @@
 from __future__ import absolute_import
+
+from .group import *  # NOQA
```

Comment:
was going to rename things on the backend too, but I think I'm going to wait since its kind of annoying having differing table names from model names